scopes/generator/generator/component-generator.ts

```diff
@@ -28,6 +28,10 @@ export type GenerateResult = {
   envId: string;
   envSetBy: string;
   packageName: string;
+  isApp?: boolean;
+  isEnv?: boolean;
```

Comment:
Can they both be true at the same time? Maybe it should be `type: 'app' | 'env' | 'component'` instead

Reply:
It might be a case that you have an app inside your env.
(not sure if I want to support it)
I'll re-think about it, maybe I'll change it.